packages/block-library/src/navigation/edit/responsive-wrapper.js

```diff
@@ -36,16 +36,24 @@ export default function ResponsiveWrapper( {
 		'wp-block-navigation__responsive-container-open',
 		{ 'always-shown': isHiddenByDefault }
 	);
+	const plainMenuClasses = classnames( 'wp-block-navigation__plain-menu', {
+		'always-shown': isHiddenByDefault,
+	} );
 
 	const modalId = `${ id }-modal`;
 
+	const dialogProps = {
+		className: 'wp-block-navigation__responsive-dialog',
+		'aria-labelledby': `${ modalId }-title`,
+		...( isOpen && { role: 'dialog', 'aria-modal': true } ),
```

Comment:
If we're adding these attributes dynamically only when the modal is open, do we still need the separate set of markup for when the nav is not in overlay mode? For the front end, could we toggle them in the view script instead?